Minor fixes around integration tests

adrianriobo opened the pull request:
Fixes #4166 
Fixes #4157

adrianriobo committed:
[qe] increase default timeout for integration tests suite

Tests are having false positives due to overall ginkgo timeout default value, this commit will include a mechanism to customize the timeout value when running the tests through the container and also increses the default value to 90 minutes

Signed-off-by: Adrian Riobo Lorenzo <ariobolo@redhat.com>

adrianriobo committed:
[qe] ensure all crc start commands set the bundle if it is passed through the params

integration tests offers a parameter to setup a custom bundle. Previous code made use of the custom bundle on some of the start commands but missed to use it on others. As so the tests end failing due to bundle missmatch versions. This commit refactor the method used to test the setup and start commands making sure if custom bundle parameter is passed it is used always

Signed-off-by: Adrian Riobo Lorenzo <ariobolo@redhat.com>

adrianriobo committed:
Remove @jsliacan from OWNERS file :(

Signed-off-by: Adrian Riobo Lorenzo <ariobolo@redhat.com>

adrianriobo committed:
[qe] added labelFilter parameter to allow filtering integration tests to be executed based on labels

Signed-off-by: Adrian Riobo Lorenzo <ariobolo@redhat.com>

anjannath commented:
/test integration-crc

anjannath commented on images/build-integration/lib/darwin/run.sh:
```diff
@@ -4,6 +4,7 @@
 bundleLocation=""
 targetFolder="crc-integration"
 junitFilename="integration-junit.xml"
+suiteTimeout="90m"
```
is the `images/build-integration/lib/darwin/run.sh` and `images/build-integration/lib/linux/run.sh` the same script? maybe we can use a symlink to avoid editing two files

adrianriobo replied:
done as per suggest on https://github.com/crc-org/crc/pull/4169/commits/7b60e2e8c05da57b7171d47ded1f4ced353de5f5

adrianriobo committed:
[qe] unified darwin and linux scripts to execute integration and e2e tests remotely

Both scripts were identically, to avoid maintain both of them this commit keeps the linux version and create a symlink under the darwin folder, so from now on and unless something specific to darwin is required only one script will be maintain.

Signed-off-by: Adrian Riobo Lorenzo <ariobolo@redhat.com>
